src/gui/syncrunfilelog.cpp

```diff
@@ -81,38 +85,50 @@ void SyncRunFileLog::logItem(const SyncFileItem &item)
         return;
     }
     const QChar L = QLatin1Char('|');
-    *_out << dateTimeStr(QDateTime::fromString(QString::fromUtf8(item._responseTimeStamp), Qt::RFC2822Date)) << L
-          << ((item._instruction != CSYNC_INSTRUCTION_RENAME) ? item.destination() : item._file + QStringLiteral(" -> ") + item._renameTarget) << L
-          << item._instruction << L
-          << item._direction << L
-          << L
-          << item._modtime << L
-          << item._etag << L
-          << item._size << L
-          << item._fileId << L
-          << item._status << L
-          << item._errorString << L
-          << item._httpErrorCode << L
-          << item._previousSize << L
-          << item._previousModtime << L
-          << item._requestId << L
-          << endl;
+    QString tmp;
```

Comment:
A lambda might be more suitable, because `tmp` does not leak into this scope. An alternative would be a dedicated method/function.